src/citrine/resources/ingredient_run.py

```diff
@@ -62,6 +62,8 @@ class IngredientRun(DataConcepts, Resource['IngredientRun'], TaurusIngredientRun
 
     _response_key = TaurusIngredientRun.typ  # 'ingredient_run'
 
+    _client_keys = ["audit_info"]
```

Comment:
The root is `DataConcepts`, which is extended by `Property`, `Parameter`, and `Condition`, none of which have `audit_info`.

Reply:
I believe my suggestion is for it to be put in `BaseObject` in taurus.

Reply:
sidenote -- do you know what the point of having attributes extend data concepts is?

Reply:
It's because attributes contain attribute templates, so they need the same complicated `build` logic in order to properly serialize and deserialize those.
Conversely, a `Value` doesn't need to extend `DataConcepts` or have a Citrine-Python implementation, because it's just a simple dictionary without anything complicated in its fields.